cdk/lib/Airdrop/airdrop-stack.ts

```diff
@@ -145,6 +145,8 @@ export class AirdropStack extends cdk.Stack {
       sources: [s3deploy.Source.asset(path.join(buildOutputDir, "assets"))],
       destinationBucket: myBucket,
       distribution: distribution,
+      prune:true,
+      retainOnDelete: false,
```

Comment:
Is there any backup of the s3 bucket? I would rather handle this manually.

Reply:
This bucket is dumping the zip files of the build, the actual site is defined on variable `myBucket`